bugsnag/src/test/java/com/bugsnag/AppenderTest.java

```diff
@@ -144,10 +144,10 @@ public void testBugsnagConfig() {
         assertEquals("gradleTask", config.appType);
         assertFalse(config.shouldAutoCaptureSessions());
 
-        assertEquals(2, config.filters.length);
-        ArrayList<String> filters = new ArrayList<String>(Arrays.asList(config.filters));
-        assertTrue(filters.contains("password"));
-        assertTrue(filters.contains("credit_card_number"));
+        assertEquals(2, config.redactedKeys.length);
+        ArrayList<String> redactedKeys = new ArrayList<String>(Arrays.asList(config.redactedKeys));
+        assertTrue(redactedKeys.contains("password"));
+        assertTrue(redactedKeys.contains("credit_card_number"));
```

Comment:
It would be worth adding a few tests to verify the expected interplay between `filters` and `redactedKeys`

Reply:
I've added functionality so that when both are called, setRedactedKeys creates a temp array to append the filters to the redactedKeys config option to avoid overwriting.